icu4c/source/test/cintltst/cloctst.c

```diff
@@ -2564,7 +2564,7 @@ static void TestCanonicalizationBuffer(void)
         return;
     }
 
-    if (reslen != len) {
+    if (reslen != (int32_t)len) {
```

Comment:
Should this be
```suggestion
    if (reslen != static_cast<int32_t>(len)) {
```
We are not really consistent in this codebase, but static_cast<int32_t> seems to outnumber (int32_t) nearly 3-to-1.

Reply:
C code must use C-style casts, right?

Reply:
Oh. Right, C exists. Nevermind.